app/components/publications/urgency-level-checkbox.js

```diff
@@ -0,0 +1,42 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import CONFIG from 'frontend-kaleidos/utils/config';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency-decorators';
+
+
+export default class UrgencyLevelCheckboxComponent extends Component {
+  @tracked urgencyLevels = null;
+  @tracked checkboxValue;
+
+  @service store;
+
+
+  constructor() {
+    super(...arguments);
+    this.loadData.perform();
+  }
+
+  @task
+  *loadData() {
+    this.urgencyLevels = yield this.store.query('urgency-level', {});
+    this.initializeUrgency(this.args.urgencyLevel);
```

Comment:
this.args.urgencylevel is een promise, die moet je dus ook yielden (of beter inladen in de parent route/component door bvb de data loading in de route uit te breiden)